wdl/Module00c.wdl

```diff
@@ -140,11 +142,13 @@ workflow Module00c {
 
     # Runtime parameters
     String sv_base_mini_docker
+    String sv_base_docker
```

Comment:
Just remembered to remind you to update the gCNV model for the single-sample run

Reply:
I ran the single-sample pipeline. There were some problems with the bincov matrix which I corrected. It ran through until Module05_06 where it crashed in CleanVcf1a due to my not having rebuilt the sv-pipeline docker after Chris and I made incompatible changes to it (the WDL is looking for the output "includelist.txt" but my old docker is providing whitelist.txt). I'm rebuilding it now because it sounds like there's immediate interest in running this.